### ISSUE TITLE
integration etcd startup is flaky

### ISSUE THREAD

**Issue by smarterclayton** [GoogleCloudPlatform/kubernetes#1086]
This should never happen - it seems to imply that etcd is now unreachable, even though for the integration test to succeed it would have to be running.

```
The command "PATH=$HOME/gopath/bin:$PATH ./hack/test-cmd.sh" exited with 0.
14.75s$ PATH=$HOME/gopath/bin:$PATH ./hack/test-integration.sh
etcd: {"action":"get","node":{"key":"/","dir":true}}
Integration test cases ...
ok  	github.com/GoogleCloudPlatform/kubernetes/test/integration	4.027s	coverage: 83.3% of statements
Integration scenario ...
I0828 17:37:08.591096 08378 integration.go:90] Creating etcd client pointing to [http://localhost:4001]
F0828 17:37:09.596388 08378 integration.go:99] Unable to list root etcd keys: 501: All the given peers are not reachable (Tried to connect to each peer twice and failed) [0]
etcd still running
The command "PATH=$HOME/gopath/bin:$PATH ./hack/test-integration.sh" exited with 255.
Done. Your build exited with 1.
```

**Comment by ixdy** [comment]
This issue hasn't been updated in a while - has this still been a problem?

**Comment by smarterclayton** [comment]
I haven't seen it on travis in a while, it may have been resolved by 2.0.  However, no proof that it has.

**Comment by gmarek** [comment]
Yeah. I tried to reproduce it locally by running multiple (20) times, ale gone through without this bug. I also looked at all travis failures in the last 24 hours. It never happened. I guess we can assume it's fixed.
